rollup.config.js

```diff
@@ -23,14 +23,13 @@ function buildConfigBuilder({ name, input, dist = "dist" }) {
     format,
     transpiled = true,
     minified = false,
-    includeExtension = true,
-    extension = format,
+    extension = "",
     sourceMap = false
   }) => {
     function buildFileName() {
-      return `${name}${includeExtension ? `.${extension}` : ""}${
-        minified ? ".min" : ""
-      }.js`;
+      return `${name}.${format === "es" ? "esm" : format}${
+        !!extension ? `.${extension}` : ""
+      }${minified ? ".min" : ""}.js`;
```

Comment:
This is going to change the way extensions are generated, and so it will break our CDNs links. Check the `package.json` file for more details about that.

I liked the idea of refactoring this `includeExtension` logic, but I think we shold keep the generated names the same as before.